src/Jira/Api/Client/CurlClient.php

```diff
@@ -117,12 +120,12 @@ public function sendRequest(
 				curl_setopt($curl, CURLOPT_POSTFIELDS, json_encode($data));
 			}
 		}
-		elseif ( $method == 'PUT' ) {
-			curl_setopt($curl, CURLOPT_CUSTOMREQUEST, 'PUT');
+		elseif ( $method == 'PUT' || $method == 'DELETE' ) {
```

Comment:
That thought have crossed my mind, but I decided not to take that patch considering much more actions involved from my side.

Since you've asked I've created #115 now and once merged I can rebase this PR on top of it.